lib/engine/game/g_18_mex/step/buy_sell_par_shares.rb

```diff
@@ -17,6 +17,7 @@ def can_sell?(entity, bundle)
           end
 
           def can_gain?(entity, bundle, exchange: false)
+            return if bundle.owner&.player?
```

Comment:
Should the !bundle check be before this line?

Reply:
yes